sql/session.proto

```diff
@@ -42,4 +42,10 @@ message Session {
   // Indicates that the above transaction is mutating keys in the
   // SystemDB span.
   optional bool mutates_system_db = 4 [(gogoproto.nullable) = false, (gogoproto.customname) = "MutatesSystemDB"];
+  oneof timezone {
+    // A time zone, local, or default (same as local).
```

Comment:
Same as whose local? I don't think we should ever use the server's local time for anything; it should either be a client-supplied timezone or UTC.